xinference/core/model.py

```diff
@@ -45,6 +45,33 @@
 T = TypeVar("T")
 
 
+def request_limit(fn):
+    async def wrapped_func(self, *args, **kwargs):
```

Comment:
The `request_limit` decorator is only available for the ModelActor, can you add some comments?